packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSliderForm.component.test.js

```diff
@@ -60,33 +64,40 @@ describe('BadgeSliderForm', () => {
 			operator: { name: 'equals' },
 		};
 		// When
-		const wrapper = mount(
+		render(
 			<BadgeFacetedProvider value={badgeFacetedContextValue}>
 				<BadgeSliderForm {...props} />
 			</BadgeFacetedProvider>,
 		);
 		// Then
-		expect(wrapper.getElement()).toMatchSnapshot();
+		const slider = screen.getByRole('slider');
+		expect(slider).toHaveAttribute('aria-valuenow', '0');
+		expect(slider).toHaveAttribute('aria-valuemin', '0');
+		expect(slider).toHaveAttribute('aria-valuemax', '100');
+		expect(slider.parentElement).toHaveClass('theme-tc-slider-rc-slider--track-equals');
 	});
 
 	it('should mount a default badge in edit mode', () => {
 		// Given
 		const props = {
 			id: 'customId',
 			onSubmit: jest.fn(),
+			value: 43,
 			feature: 'quality',
 			t: getDefaultT(),
 			onChange: jest.fn(),
 		};
 		// When
-		const wrapper = mount(
+		render(
 			<BadgeFacetedProvider value={badgeFacetedContextValue}>
 				<BadgeSliderForm {...props} />
 			</BadgeFacetedProvider>,
 		);
 
-		wrapper.find('.tc-badge-value-unit').first().simulate('click');
-		expect(wrapper.getElement()).toMatchSnapshot();
+		window.HTMLFormElement.prototype.requestSubmit = jest.fn();
```

Comment:
Should we put it back to its original value after the test ?

Reply:
it works without it so let's remove it. Don't remember why I need this during the rewrite :thinking: